source/standards/securing-services.html

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
This file needs to be named `securing-services.html.md.erb` to be rendered correctly 😖